src/format/KdbxXmlReader.cpp

```diff
@@ -188,6 +188,15 @@ QString KdbxXmlReader::errorString() const
     return QString();
 }
 
+bool KdbxXmlReader::isTruthyAttribute(const QString& name)
```

Comment:
I wouldn't call this `isTruthyAttribute`. Call is `isTrueValue` and return true if the string matches any of the True values. Move the attribute logic outside, so we can reuse the method for boolean elements, not just attributes.